upgrade/upgrade-2.0.0.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * 2017 thirty bees
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@thirtybees.com so we can send you a copy immediately.
+ *
+ *  @author    thirty bees <modules@thirtybees.com>
+ *  @copyright 2017 thirty bees
+ *  @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ */
```

Comment:
We have 2018 in the Christian calendar :-)

Here's a template for such headers: https://github.com/thirtybees/thirtybees/blob/1.0.x/tools/templates/header.php-js-css.tb.module

Reply:
:) i will update it now